### DISCUSSION TITLE
Is it possible to add a function with a set time？

### DISCUSSION THREAD

**Discussion by Dayu0815** [AutomaApp/automa#82] [Q&A]
Hello! Is it possible to add a function with a set time, which can automatically execute Workflows at a specified time without opening the browser?

**Comment by Kholid060** [comment]
It's not possible to execute workflow without opening the browser, because all the extension components rely on the browser

**Reply by Dayu0815** [reply]
If you must open the browser, is it recommended to add a function with a set time?

**Reply by Kholid060** [reply]
If you mean to execute workflow at a specific time, you can do that by clicking the pencil button of the **Trigger block** and then selecting "On specific date"
![image](https://user-images.githubusercontent.com/22908993/141677062-fd3ae228-2783-4524-9201-8e210a544854.png)

**Reply by Dayu0815** [reply]
Yes, it is this function. May I ask if the adjustment is not on a certain day, can it be set to execute the workflow within a certain period of time?

**Reply by Dayu0815** [reply]
Or, is it possible to set the specified workflow to be executed every 24 hours?

**Reply by Kholid060** [reply]
> May I ask if the adjustment is not on a certain day

This feature is in [to do](https://github.com/Kholid060/automa/projects/1#card-72506468) list, it will available in the future update

**Reply by Dayu0815** [reply]
Okay, thank you for your reply, and look forward to your latest update as soon as possible

**Reply by Dayu0815** [reply]
Hello! Can "Automa" be used in conjunction with shortcuts in iPhones?

**Reply by Dayu0815** [reply]
Hello!
I would like to ask if it is possible to adjust the trigger conditions of "on specific date" and "when visit a website". Only one-time triggers are accepted, and the workflow after the second trigger will not be started.

**Reply by Dayu0815** [reply]
Hello! Has there been an update to a "Trigger event" feature recently? How to use it?

**Reply by Kholid060** [reply]
There's no update for the "Trigger event" recently, it might be available in the next update.

**Reply by Dayu0815** [reply]
Okay, thank you for your reply, thank you for your hard work

**Reply by Dayu0815** [reply]
Good morning!
May I ask how the "Use regex" function should be used?

**Reply by Kholid060** [reply]
It's to tell the workflow whether the value that you inputted is treated as a [regex pattern](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#writing_a_regular_expression_pattern) or not.

**Reply by Dayu0815** [reply]
Thank you for your explanation, and I would like to ask, for each function of "Automa", is it possible to set a location so that we can click to download the operation manual or read the function introduction?

**Reply by Dayu0815** [reply]
Hello!
For the "Take screenshot" function, is it possible to set the photo storage location and the editing method of the photo file name in advance to speed up the processing process?

**Reply by Kholid060** [reply]
It's not possible to set the storage location of the photo

**Reply by Dayu0815** [reply]
Thank you for adding a function with a set time. May I ask that if the browser is not opened, Workflows cannot be executed automatically, right?

**Reply by Dayu0815** [reply]
Is it possible to open the browser through the mobile phone, or under the IOS system environment, it can also automatically execute Workflows normally?

**Reply by Kholid060** [reply]
> Thank you for adding a function with a set time. May I ask that if the browser is not opened, the workflow cannot be executed automatically, right?

Yes, the workflow won't be executed.

> Is it possible to open the browser through the mobile phone, or under the IOS system environment, it can also automatically execute Workflows normally?

If you open the browser where you create the workflow in. It will run normally

**Reply by Dayu0815** [reply]
Hello! Excuse me!
When we click on the "workflows window", can we set it to go forward and backward, and adjust the display order of each workflow freely up and down?
